docs/source/algorithms/SingleCrystalDiffuseReduction-v1.rst

```diff
@@ -0,0 +1,221 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+Developed for CORELLI but should work on any instrument. This workflow
+algorithm loops over a series of runs combining them with correct
+normalisation, subtract the background and apply symmetry. The
+resulting workspace is a :ref:`MDHistoWorkspace <MDHistoWorkspace>`
+containing a volume of scattering.
+
+The input filename follows the syntax from
+:py:obj:`MultipleFileProperty <mantid.api.MultipleFileProperty>`
+
+This workflow makes use of :ref:`ConvertToMD <algm-ConvertToMD>` and
+:ref:`MDNormSCD <algm-MDNormSCD>` so these should be reviewed to
+better understand all the options. An example of creating the Solid
+Angle and Flux workspaces are included in :ref:`MDNormSCD
+<algm-MDNormSCD>`.
+
+The resulting workspaces can be saved and loaded with :ref:`SaveMD
+<algm-SaveMD>` and :ref:`LoadMD <algm-LoadMD>` respectively.
+
+Masking
+#######
+
+The mask from the solid angle workspace is copied to the
+data. Additional masking is provided by :ref:`MaskBTP <algm-MaskBTP>`
+using the Bank, Tube and Pixel parameters.
```

Comment:
You should mention that this mask is applied to all data and it's only one mask. So you cannot do bank 23 and only tube 3 from bank 24